client/src/www/app/outline_server_repository/config.ts

```diff
@@ -90,6 +107,9 @@ export function parseTunnelConfig(
     );
   }
 
+  // TODO(fortuna): stop converting to the Go format. Let the Go code convert.
```

Comment:
I cannot add review feedback on unchanged lines, but the comment at Line 92 is not accurate any more:

> `This is used by the server to parse the config fetched from the dynamic key.`

It parses both static key and the content of dynamic key now.

Reply:
The comment is correct. The config fetched from the dynamic key may be a ss:// url. But I added a clarification.